src/pages/fresh_install/FreshInstallPage.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import '../fresh_install/FreshInstallPage.css';
+import path4 from "../../images/path4.svg";
+import classNames from 'classnames';
+
+export default class FreshInstallPage extends React.Component<{}, { selectedTemperatureScale: string }> {
+
+    handleButtonClick = (selectedTemperatureScale: 'C' | 'F') => {
+        if (this.state === null || !(this.state.selectedTemperatureScale === selectedTemperatureScale)) {
+            this.setState({
+                selectedTemperatureScale
+            });
+
+            localStorage.setItem('temperatureScale', selectedTemperatureScale);
+        }
+    }
+
+    public moveToLocation() {
+        console.log("Moving");
+    }
+
+    renderCelsiusButton = () => {
+        const isClicked = this.state !== null && this.state.selectedTemperatureScale === 'C';
+        const divClasses = classNames({
+            ButtonActive: isClicked,
+            ButtonCelsius: true,
```

Comment:
For those values that are always `true`, we can move the class name outside the json property, like:
```js
const divClasses = classNames('ButtonCelsius', {
    ButtonActive: isClicked,
    Button: !isClicked
});
```

For more info: https://github.com/JedWatson/classnames#usage